assets/css/style.css

```diff
@@ -256,6 +256,7 @@ button:hover {
   width: 129px;
   height: 48px;
   background: #ff6b00;
+  text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.5);
```

Comment:
## Color Contrast: 
The color contrast between the text and background meets the minimum requirement, however, some elements could benefit from a higher contrast ratio. 
Changes suggested.
We fixed this.